pages/services-clearing/index.js

```diff
@@ -0,0 +1,128 @@
+import { makeStyles, useTheme } from '@material-ui/core/styles';
```

Comment:
Pls fix the tab spacing in this file (it's set to `4` here, and `2` in the other files)